sherpa/astro/ui/utils.py

```diff
@@ -3491,7 +3580,10 @@ def get_error(self, id=None, filter=False, bkg_id=None):
         return d.get_error(filter, self.get_stat().calc_staterror)
 
     # DOC-NOTE: also in sherpa.utils
-    def get_indep(self, id=None, filter=False, bkg_id=None):
+    def get_indep(self,
+                  id: Optional[IdType] = None,
+                  filter: bool = False,
+                  bkg_id: Optional[IdType] = None):
```

Comment:
Should the return value be typed? (Same for a few other modified functions below)

Reply:
See https://github.com/sherpa/sherpa/pull/2114#discussion_r1732781845